packages/sui-widget-embedder/README.md

```diff
@@ -134,6 +134,34 @@ $ sui-widget-embedder generate <widgetName> -E 'expression'
 
 Note that the quotes here are not 'optional' you can add an expresion without quotes off course but you'll need to escape all the chars that are interpretable by the terminal.
 
+## Hot reloading
+
+In order to be able to use webpack's hot reloading feature, you'll need to use the hot reloader module inside your widget.js like this:
+
+```js
+import {hot} from 'react-hot-loader'
+
+const YourAwesomeWidget = () => ...
+
+export default hot(module)(YourAwesomeWidget) // don't worry about "module", it will work thanks to webpack
+```
+
+## Load widget on the fly without proxy
+
+In case you want to load your widget in a site, you need to follow these steps:
```

Comment:
Maybe you must say that this is the prefer case for site with user & pass.

Dashboards or privite user zone